schema/build-config-schema.js

```diff
@@ -498,6 +498,24 @@ for (const rule of rules) {
         }
       };
       break;
+    case "MD054":
+      scheme.properties = {
+        "style": {
+          "description": "Link or image style should be consistent",
+          "type": "string",
+          "enum": [
+            "mixed",
+            "autolink_only",
+            "inline_only",
+            "reference_only",
+            "inline_or_reference",
+            "inline_or_autolink",
```

Comment:
Super duper nitpick: Please move this a line up so it comes before inline_or_reference

Reply:
Sure, thanks!